src/imageproxy.rs

```diff
@@ -192,6 +193,42 @@ impl TryFrom<ImageProxyConfig> for Command {
     }
 }
 
+
+///Trying
+#[derive(Debug, serde::Deserialize)]
+pub struct LayersInfo {
+    ///Placeholder
+    pub Success: String,
```

Comment:
I'd like to understand this one - we already have a mechanism to return errors in the protocol.  This member doesn't seem to be referenced in the code here.

If we can drop it, it would seem to simplify things.